scripts/colab_fuzzy_dedup_study.py

```
@@ -0,0 +1,458 @@
+        cache_kwargs=io_kwargs,
+        write_kwargs=io_kwargs,
+    )
+    identify.run()
+    identify_sec = time.time() - st
+
+    duplicate_ids_path = fuzzy_output_dir / "FuzzyDuplicateIds"
+    cc_path = cache_dir / "ConnectedComponentsStage"
+
+    duplicate_count = 0
+    if duplicate_ids_path.exists():
+        duplicate_count = int(len(pd.read_parquet(duplicate_ids_path)))
```

Ruff (0.15.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)